Dockerfile

```diff
@@ -5,12 +5,12 @@ ARG MAVEN_OPTS
 EXPOSE 8080
 
 # Update packages and install tools
-RUN apt-get update -y && \
-    apt-get install -y --no-install-recommends \
-      wget unzip curl \
+RUN apt-get update -y
+RUN apt install -y --no-install-recommends \
+      wget unzip curl net-tools \
```

Comment:
Waarom voeg je net-tools toe?

Reply:
Ik had een aantal keer netstat nodig. Dus het leek me handig voor future debugging om het toe te voegen. In production natuurlijk niet nodig, maar het lijkt me ook geen probleem om netstat available te hebben.